crates/swc_css_parser/tests/fixture/function/calc/span.rust-debug

```diff
@@ -11462,36 +11342,6 @@
     :                       ^^^^^^^^^^^^
     `----
 
-  x CalcSum
-    ,-[$DIR/tests/fixture/function/calc/input.css:88:5]
- 88 | width: round(to-zero, var(--width), 50px);
-    :                       ^^^^^^^^^^^^
-    `----
-
-  x CalcProductOrOperator
-    ,-[$DIR/tests/fixture/function/calc/input.css:88:5]
- 88 | width: round(to-zero, var(--width), 50px);
-    :                       ^^^^^^^^^^^^
-    `----
-
-  x CalcProduct
-    ,-[$DIR/tests/fixture/function/calc/input.css:88:5]
- 88 | width: round(to-zero, var(--width), 50px);
-    :                       ^^^^^^^^^^^^
-    `----
-
-  x CalcValueOrOperator
-    ,-[$DIR/tests/fixture/function/calc/input.css:88:5]
- 88 | width: round(to-zero, var(--width), 50px);
-    :                       ^^^^^^^^^^^^
-    `----
-
-  x CalcValue
-    ,-[$DIR/tests/fixture/function/calc/input.css:88:5]
- 88 | width: round(to-zero, var(--width), 50px);
-    :                       ^^^^^^^^^^^^
-    `----
-
```

Comment:
Strange change

Reply:
I think this change is similar to that in AST output: it isn't a `CalcSum` any more, instead, it's `Function` directly. Previously, it's a `CalcSum` that wraps a `Function`.

Reply:
It should 

> it's a CalcSum that wraps a Function

https://www.w3.org/TR/css-values-4/#calc-syntax 

That is why we have https://github.com/swc-project/swc/pull/4713#discussion_r878707487 problem

Reply:
Looks like we need to use `try_parse_variable` in calc functions too, regarding to syntax `var` is general syntax for any `calc-value`

Reply:
It's already used in math functions: https://github.com/g-plane/swc/blob/67c4aa1efbe86aa440927a8661bf56ebfb7432ac/crates/swc_css_parser/src/parser/value/mod.rs#L267 .